src/js/model/primitives/Guide.js

```diff
@@ -1,9 +1,18 @@
+
 var Primitive = require('./Primitive'),
     model  = require('../'),
     lookup = model.primitive;
 
 var GTYPES = {AXIS: 1, LEGEND: 2},
     ORIENT = {x: 'bottom', y: 'left'};
+/**
+* Guide
+* @constructor
+* @param {string} gtype
+* @param {string} type
+* @param {string} scale
+*
+*/
```

Comment:
Awesome!

Speaking of the sublime plugin, @arvind if you use Sublime, the DocBlockr extension is great: it auto-completes and lets you tab through a snippet.
![dockblockr](https://cloud.githubusercontent.com/assets/442115/13444815/7ff1e628-dfd6-11e5-96bb-eda3fba19c8d.gif)